pkg/i18n/english.go

```diff
@@ -932,8 +942,11 @@ func EnglishTranslationSet() TranslationSet {
 		// the actual view is the extras view which I intend to give more tabs in future but for now we'll only mention the command log part
 		LcOpenExtrasMenu:                    "open command log menu",
 		LcShowingGitDiff:                    "showing output for:",
+		LcCopyCommitDiffToClipboard:         "copy commit diff to clipboard",
```

Comment:
I think that we can spare ourselves some repetition in the menu items:
![image](https://user-images.githubusercontent.com/8456633/160014016-5f3db975-0b16-4b1d-af31-cbd7d46e7c79.png)

How about having a title that says 'Copy to clipboard' and then the options are 'commit URL', 'commit SHA', 'commit diff', 'commit message'?

Reply:
Also, I'd have the 'SHA' be the first option. I think that the thing you copy by pressing '<c-o>' should be at the top of the menu. Part of the reason for this is that it actually makes it easier to create a generic 'clipboard' controller down the line where it can take a list of menu items and then add both a menu-based on direct clipboard keybinding